### PR TITLE
Fix IMA guard in paged MQA logits scheduler

### PR DESCRIPTION
**Pull request by nvjullin**
Cherry-picked from https://github.com/deepseek-ai/DeepGEMM/pull/342

### CODE REVIEW

**Comment by b8zhong** [comment]
@nvjullin Can you change the target to be dev branch? https://github.com/sgl-project/DeepGEMM/tree/dev
